### DISCUSSION TITLE
Is there any way to force dark mode by default?

### DISCUSSION THREAD

**Discussion by cguy** [histoire-dev/histoire#324] [Q&A]
Hello :) 

I am starting to use Histoire for Svelte (because Storybook doesn't let me yet to use nodejs18). My application has only a dark custom theme, so I would like to force Histoire to be in darkmode as well (and, if possible, forbid light mode).

Thanks !

**Comment by Akryum** [comment]
Hey! That looks like it could be a valid feature request. Could you open on in the `Issues` tab? Thanks! :pray:
